### DISCUSSION TITLE
are there any plans to update some libraries

### DISCUSSION THREAD

**Discussion by eliabexp** [anyproto/contributors#208] [Anytype apps]
Hi, I use Anytype for about a year and I love this app.
I pretend to contribute with the code, but I would like to know if there's a reason for Anytype PC client still use some deprecated versions of libraries like react and sass

And if we contributors could update the code for the new versions of react, pretending to improve the code quality.

**Comment by fuksman** [comment]
Hello @eliabexp. 
Unfortunately, it is currently impossible to update versions of these libraries just due to other dependencies in the project. 
We do not recommend you to make such contributions; it is better to focus on other improvements :)
